as-ap-opticscorr/as_ap_opticscorr/tune/main.py

```diff
@@ -465,114 +480,125 @@ def _callback_estimate_deltatune(self, pvname, value, **kws):
 
         # Change the third bit of correction status
-        opmode = self._sync_corr
-        val = (1 if any(s != opmode for s in self._qfam_check_opmode_sts)
-               else 0)
+        opmode = _PSConst.States.SlowRefSync if self._sync_corr \
+            else _PSConst.States.SlowRef
         self._status = _siriuspy.util.update_bit(
-            v=self._status, bit_pos=2, bit_val=val)
+            v=self._status, bit_pos=2,
+            bit_val=any(s != opmode for s in self._qfam_check_opmode_sts))
         self.driver.setParam('Status-Mon', self._status)
         self.driver.updatePVs()
 
     def _callback_qfam_ctrlmode_mon(self,  pvname, value, **kws):
-        ps = pvname.split(self._PREFIX_VACA)[1]
-        if value == 1:
-            self.driver.setParam('Log-Mon', 'WARN:'+ps+' is Local.')
+        if value != _PSConst.Interface.Remote:
+            self.driver.setParam('Log-Mon', 'WARN:'+pvname+' is not Remote.')
             self.driver.updatePVs()
 
-        fam = ps.split(':')[1].split('-')[1]
-        fam_index = self._QFAMS.index(fam)
+        fam_index = self._QFAMS.index(_SiriusPVName(pvname).dev)
         self._qfam_check_ctrlmode_mon[fam_index] = value
 
         # Change the fourth bit of correction status
-        val = (1 if any(q == 1 for q in self._qfam_check_ctrlmode_mon)
-               else 0)
         self._status = _siriuspy.util.update_bit(
-            v=self._status, bit_pos=3, bit_val=val)
+            v=self._status, bit_pos=3,
+            bit_val=any(q != _PSConst.Interface.Remote
+                        for q in self._qfam_check_ctrlmode_mon))
         self.driver.setParam('Status-Mon', self._status)
         self.driver.updatePVs()
 
     def _callback_timing_state(self, pvname, value, **kws):
         if 'Quads:State' in pvname:
-            self._timing_check_config[0] = value  # Enbl
-        elif 'Quads:EVGParam' in pvname:
-            self._timing_check_config[1] = (1 if value == 1 else 0)  # Tunes
-        elif 'Quads:Pulses' in pvname:
-            self._timing_check_config[2] = (1 if value == 1 else 0)  # 1 pulse
+            self._timing_check_config[0] = (value == _TIConst.DsblEnbl.Enbl)
+        elif 'Quads:Polarity' in pvname:
+            self._timing_check_config[1] = (value == _TIConst.TrigPol.Normal)
+        elif 'Quads:Src' in pvname:
+            self._timing_check_config[1] = (value == self._tunsi_src_idx)
+        elif 'Quads:NrPulses' in pvname:
+            self._timing_check_config[2] = (value == 1)  # 1 pulse
         elif 'Quads:Duration' in pvname:
-            self._timing_check_config[3] = (1 if value == 0.15 else 0)  # 150us
-        elif 'TunesMode' in pvname:
-            self._timing_check_config[4] = (1 if value == 3 else 0)  # External
-        elif 'TunesDelay' in pvname:
-            self._timing_check_config[5] = (1 if value == 0 else 0)  # 0s
+            self._timing_check_config[3] = (value == 0.15)  # 150us
```

Comment:
Ana, the `Duration` PV from timing is in `us` now. You should change the comparison here to `value == 150`